dealership/contracts.py

```diff
@@ -1,12 +1,54 @@
+from .vehicles import Car, Truck, Motorcycle
+
+
 class Contract(object):
-    pass
+    def __init__(self, vehicle, customer):
```

Comment:
The `Contract` class does not use raise any `NotImplemented` exceptions. This means users will be able instantiate the `Contract` class if desired. This is really a matter of how you want your code to work. If you're okay with people on the outside being able to use the `Contract`class alone, then that's completely fine. However, if you want this class to be more "abstract", then I suggest making it not implementable.